### DISCUSSION TITLE
2022/09/15/diary-2022-sept/

### DISCUSSION THREAD

**Discussion by giscus** [DeCalvin2006/blog#8] [General]
# 2022/09/15/diary-2022-sept/

这是2020年9月份的日记，因为分开有水的嫌疑就合订了。

https://decalvin.netlify.app/2022/09/15/diary-2022-sept/

**Comment by shiziping** [comment]


**Reply by DeCalvin2006** [reply]
$SZP AK IOI$

**Comment by shiziping** [comment]
嗨害嗨
$Decalvin\ AK\ IOI!$
